Literally.hs

```diff
@@ -9,6 +9,65 @@
 {-# LANGUAGE TypeFamilies #-}
 {-# LANGUAGE UndecidableInstances #-}
 
+-- |
+-- Module      : Literally
+-- Description : Type-safe conversion of type literals into runtime values
+-- Copyright   : (c) Taylor Fausak
+-- License     : 0BSD
+-- Maintainer  : Taylor Fausak
+--
+-- Literally is a minimal Haskell library that converts type literals into values
+-- using type-safe mechanisms. It provides a single 'literal' function that
+-- leverages Haskell's type system to convert compile-time literals into runtime
+-- values with compile-time correctness guarantees.
```

Comment:
Similarly, this does not need to mention that it's a Haskell library because that's already clear from context.